### DISCUSSION TITLE
Connection between OR and ORFS

### DISCUSSION THREAD

**Discussion by lokki11199** [The-OpenROAD-Project/OpenROAD#3479] [Q&A] [answered]
Hello! I would like to ask a question about the connection between OR and ORFS. Does ORFS contain an OR app? What is the difference between OR and ORFS apart from the fact that the latest one also contains Yosys and can create a gds file, i.e. provides the entire RTL-GDSII flow. I understand that ORFS contains ready-made scripts to run the flow and only need to set some variables, but what are the global differences?
Also, I asked a question about ORFS features. I was interested in the possibility of multi-corner analysis. I was told that only OR can be configured for such analysis. Unfortunately, I haven't found a clear answer to my question in the documentation, could you please tell me how to do it or where I can find information.

Thank you for responding!

**Comment by maliberty** [comment, marked as answer]
OR is the application.  ORFS is a flow that runs yosys, OR, and klayout to implement a design.

You can find the STA manual at https://github.com/The-OpenROAD-Project/OpenSTA/blob/master/doc/OpenSTA.pdf.  OpenLane uses multi-corner analysis and can also serve as an example.
